lib/executor/ramping_arrival_rate_test.go

```diff
@@ -578,8 +578,8 @@ func sqrtRat(x *big.Rat) *big.Rat {
 	var z, a, b big.Rat
 	var ns, ds big.Int
 	ni, di := x.Num(), x.Denom()
-	z.SetFrac(ns.Rsh(ni, uint(ni.BitLen())/2), ds.Rsh(di, uint(di.BitLen())/2))
-	for i := 10; i > 0; i-- { // TODO: better termination
+	z.SetFrac(ns.Rsh(ni, uint(ni.BitLen())/2), ds.Rsh(di, uint(di.BitLen())/2)) //nolint:gosec
+	for i := 10; i > 0; i-- {                                                   // TODO: better termination
```

Comment:
Why indent is so weird? :thinking:

Reply:
Do you want me to change this @olegbespalov ?

Reply:
Whatever you prefer, I don't mind having this even if it looks weird.

Reply:
I will leave it for now then :sweat_smile: